Exiled.API/Features/Map.cs

```diff
@@ -139,7 +167,14 @@ public static void Broadcast(ushort duration, string message, global::Broadcast.
         /// <returns>Returns the spawn point <see cref="Vector3"/>.</returns>
         public static Vector3 GetRandomSpawnPoint(this RoleType roleType)
         {
-            GameObject randomPosition = Object.FindObjectOfType<SpawnpointManager>().GetRandomPosition(roleType);
+            GameObject randomPosition;
+
+            if (spawnpointManager == null)
```

Comment:
Don't assume that this works as expected, because after the scene change, the cached component isn't disposed and when we try to get a GameObject of that component we'll get a NRE.

You've to reset that component every restart round.

Reply:
Didn't know that! Will revert it back if I don't find any workaround.